docker/nvidia/README.md

```diff
@@ -0,0 +1,14 @@
+# Setting up your DTK development container to leverage NVIDIA GPUs on the host
+```bash
+[HOST]$ cd docker/nvidia
+[HOST]$ ./setup_nvidia_docker_compose.py
+[HOST]$ docker-compose build
+[HOST]$ docker-compose -p $USER up -d
+[HOST]$ docker exec -it <container_name> bash
```

Comment:
Not necessarily.  The container name will look like `<prefix>_<service_name>_<suffix>`
I recommend to use the `-p` option to set the prefix.  Here it would be your username.  If you don't provide it, it will be the directory that contains the Compose file (i.e. "docker" I suppose).
The service name is "dtk_dev" a priori.
In most cases the suffix will be "1".